realsense_camera/CMakeLists.txt

```diff
@@ -110,6 +110,17 @@ if (CATKIN_ENABLE_TESTING)
   add_dependencies(tests_rgbd_topics ${catkin_EXPORTED_TARGETS})
 endif()
 
+set (SCRIPT_DIR ${CATKIN_DEVEL_PREFIX}/lib/${PROJECT_NAME})
+if (CATKIN_ENABLE_TESTING)
+  file(COPY test/files/scripts DESTINATION ${SCRIPT_DIR})
+endif()
+
+if (CATKIN_ENABLE_TESTING)
+  add_custom_target(clean-script
+    rm -rf ${SCRIPT_DIR}/scripts
```

Comment:
I want to remove the copied scripts when user clean the building environment, but not find how to remove it when run "catkin_make clean". so I add a customer target to remove them.
What it the correct method?